src/StructuredWriter.elm

```diff
@@ -34,7 +34,7 @@ type Writer
 
 asIndent : Int -> String
 asIndent =
-    flip String.repeat " "
+    \a -> String.repeat a " "
```

Comment:
Didn't see this; got changed automatically by `elm-upgrade`. Maybe something like
```elm
asIndent : Int -> String
asIndent amount =
    String.repeat amount " "
```
would be better?

Reply:
I definitely agree that that looks better :)